modules/lti/src/utils.ts

```diff
@@ -1,5 +1,75 @@
 import { parse as parseQuery } from "query-string";
+import { Track } from "./OpencastRest";
 
 export function parsedQueryString() {
     return parseQuery(window.location.search);
 }
+
+/**
+ * Returns the given string, with the first letter capitalized
+ * @param s the string to capitalize
+ */
+export function capitalize(s: string) {
+  if (typeof s !== 'string') {
+    return '';
+  }
+  return s.charAt(0).toUpperCase() + s.slice(1);
+}
+
+/**
+ * Sorts an array of tracks alphabetically by their type
+ * If tracks have the same type, they are then sorted by resolution
+ * @param tracks the tracks to be sorted
+ */
+export const sortByType = (tracks: Track[]) => {
```

Comment:
I would prefer to move this function into its own ts file. Something like trackUtils.ts. As such, utils will only contain general usable functions and track specific stuff can go into trackUtils

Reply:
I am pretty sure @KatrinIhler is not a fan of utility classes, that never been used as intended. I share her opinion.

Reply:
Ok, It is currently in a utils file, sorry for noise. But do you really want more utils files? really? :D

Reply:
Ok, It is currently in a utils file, sorry for noise. But do you really want more utils files? really? :D

Reply:
I have been summoned? :D (It's especially funny since I was just talking to Malte about his utils file. :rofl:)

I can't be bothered to look at this closely right now, so just a general request: no more utility files than absolutely necessary, and if you can name them better than something-utils, please do. :D

Reply:
The main reason for my request was because I did not want to have a utils file which contains a lot of miscellaneous function.